src/dash-table/components/CellFactory.tsx

```diff
@@ -113,12 +114,15 @@ export default class CellFactory {
             selected_cells
         );
 
+        const data_loading = (loading_state && loading_state.is_loading && (loading_state.prop_name === 'data' || loading_state.prop_name === undefined)) || false;
```

Comment:
Testing this out of multi-output with a single output, coming across a third case: `loading_state.prop_name === ''` :(
```suggestion
        const data_loading = (loading_state && loading_state.is_loading && (loading_state.prop_name === 'data' || loading_state.prop_name === '' || loading_state.prop_name === undefined)) || false;
```